tests/LexerTest.php

```diff
@@ -45,13 +45,8 @@ public function test_number_values($value, $expectedValue)
     public static function validNumberValuesProvider()
     {
         return [
-            [1, 1],
```

Comment:
Doctrine's Lexer does not allow/support integers and on `Coduo\PHPMatcher\Parser:L53` strings are always passed to `Lexer::setInput()`